features/0212-pickup/README.md

```diff
@@ -60,10 +60,37 @@ A request to have multiple waiting messages sent inside a `batch` message.
 {
     "@id": "123456781",
     "@type": "https://didcomm.org/messagepickup/1.0/batch-pickup",
-    "batch_size": 10
+    "batch_size": 10,
+    "~timing": {
+      "delay_milli": 3000
+    }
 }
 ```
 
+`batch_size` is count of messages that **recipient** wants to retrieve
+
+#### message timings 
+[feature-0032](https://github.com/Purik/aries-rfcs/tree/0212_pickup/features/0032-message-timing)
+
+`~timings.delay_milli` *(optional)* is timeout in milliseconds _recipient_ is ready to wait. For example, if **recipient** has set
```

Comment:
On the Aries call, it was proposed that a little more reasoning of why this is useful would be good.  Something:

- Allows the **recipient** to indicate what their HTTP timeout is on a request, so that the message holder can wait for additional messages for a period of time and still respond before the request is timed out by the recipient.